spec/support/spec/rubinhab.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
```

Comment:
Typo on file name?

Reply:
Fixed.  The name of the file actually isn't significant as every file under `spec/support/spec` is loaded.